### DISCUSSION TITLE
I share a video of an app which created with uno

### DISCUSSION THREAD

**Discussion by Daoting** [unoplatform/uno#4962] [Show and tell]
https://user-images.githubusercontent.com/29876815/104289441-1a9a3800-54f4-11eb-8e4d-d5159b95ecb9.mp4

https://user-images.githubusercontent.com/29876815/104289480-27b72700-54f4-11eb-8432-a4a3e3cfa1ad.mp4

https://user-images.githubusercontent.com/29876815/104289518-300f6200-54f4-11eb-905c-87748fda4fb0.mp4

**Comment by MartinZikmund** [comment]
Wow! Looks great!

**Comment by dr1rrb** [comment]
Really cool to see that! 🤩🤩🤩

**Comment by sasakrsmanovic** [comment]
Really great. I'd love to feature it on Uno site and/or blogs. Are you able to contact me at sasha@platform.uno please @Daoting

**Comment by Daoting** [comment]
Ok thanks. @sasakrsmanovic

**Comment by lindexi** [comment]
Awesome

**Comment by githubxiaoa** [comment]
666

**Comment by densen2014** [comment]
666
